### PR TITLE
added aframejs 

### PR DESCRIPTION
**Pull request by vyaspranjal33**

### CODE REVIEW

**Comment by vyaspranjal33** [comment]
list of some Websites using aframejs
https://makexhappen.com/
http://kurukshetra.org.in/

File: src/apps.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+		"aframe": {
+			"cats": [
+				"6"
+			],
+			"icon": "aframe.png",
+			"implies": "JavaScript",
+			"website": "https://aframe.io"
+		},
+{
+	"apps": {
```
Comment (a-khajelou) [inline src/apps.json]:
```
{
"apps": { 
```
must be removed , current json syntax is not correct

Comment (vyaspranjal33) [reply src/apps.json]: please do correction @a-khajelou

**Comment by vyaspranjal33** [comment]
@a-khajelou  i've corrected json syntax @AliasIO

**Comment by vyaspranjal33** [comment]
@AliasIO  i have updated aframe icon to svg file please review it and merge my PR

File: src/icons/aframe.svg [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (jvoisin) [inline src/icons/aframe.svg]: Do feel free to remove the comment :)

File: src/apps.json [hunk 3]
```diff
@@ -1,5 +1,14 @@
 {
 	"apps": {
+		"aframe": {
+			"cats": [
+				"6"
+			],
+			"icon": "aframe.svg",
+			"implies": "JavaScript",
```
Comment (jvoisin) [inline src/apps.json]: This is wrong: javascript is a "feature" from your browser, not something that is present due to the usage of `aframe.js`.

Comment (jvoisin) [inline src/apps.json]: You need to add a way to detect `aframs.js.

For example, if you take a look at the [hello world example]( https://aframe.io/aframe/examples/boilerplate/hello-world/ ), you can see that the `src` attribute of the `script` tag contains the string `aframe-master.js`. maybe it's worth investigating if it's always named like this.

Comment (jvoisin) [reply src/apps.json]: There is also the presence of the `AFRAMD` and `aframeStats` javascript variables.

Comment (vyaspranjal33) [reply src/apps.json]: that's what m saying i don't know what to add & how to add to detect aframejs @jvoisin

**Comment by vyaspranjal33** [comment]
@AliasIO will you let me know why aframejs pr isn't merged yet

**Comment by AliasIO** [comment]
I haven't merged it because without any patterns it doesn't do anything.

**Comment by jvoisin** [comment]
I already said that on a previous review:

> You need to add a way to detect aframs.js.

> For example, if you take a look at the hello world example, you can see that the src attribute of the script tag contains the string aframe-master.js. maybe it's worth investigating if it's always named like this.

> There is also the presence of the AFRAMD and aframeStats javascript variables.

File: src/apps.json [hunk 4]
```diff
@@ -4,6 +4,19 @@
 			"cats": [
 				"6"
 			],
+       "env": "aframe",
+  "headers": {
+    "X-Powered-By": "Aframe"
+  },
+  "html": "<link[^>]aframe\\.css",
+  "implies": "PHP\\;confidence:50",
```
Comment (jvoisin) [inline src/apps.json]: No, aframejs doesn't imply PHP at all.

File: src/apps.json [hunk 5]
```diff
@@ -4,6 +4,19 @@
...
+  "meta": {
+    "generator": "(?:Application Name|Alternative Application Name)"
```
Comment (jvoisin) [inline src/apps.json]: This is a placeholder from aframe, it won't be used in production I think.

File: src/apps.json [hunk 6]
```diff
@@ -4,6 +4,19 @@
...
+  },
+  "script": "aframe.min-([0-9.]+)\\.js\\;confidence:50\\;version:\\1",
```
Comment (jvoisin) [inline src/apps.json]: Why a 50% confidence? I think you can remove this :)

File: src/apps.json [hunk 7]
```diff
@@ -4,6 +4,19 @@
...
+  "url": ".+\\.aframe\\.io",
```
Comment (jvoisin) [inline src/apps.json]: This is wrong, url is for the __current__ url.

File: src/apps.json [hunk 8]
```diff
@@ -4,6 +4,19 @@
 			"cats": [
 				"6"
 			],
+       "env": "aframe",
+  "headers": {
+    "X-Powered-By": "Aframe"
```
Comment (jvoisin) [inline src/apps.json]: I don't think that this actually exists…

File: src/apps.json [hunk 9]
```diff
@@ -4,6 +4,19 @@
...
+  },
+  "html": "<link[^>]aframe\\.css",
```
Comment (jvoisin) [inline src/apps.json]: It would be nice to narrow this a bit more if possible.

File: src/apps.json (hunk 4 above)
Comment (vyaspranjal33) [reply src/apps.json]: you're seeing wrong block please recheck because i don't found these php line @jvoisin  can you please make the remaining changes

**Pull request closed without merging**